include/verilated_vpi.cpp

```diff
@@ -565,8 +585,14 @@ class VerilatedVpiImp final {
     }
     static void cbTimedRemove(uint64_t id, QData time) {
         // Id might no longer exist, if already removed due to call after event, or teardown
-        const auto it = s().m_timedCbs.find(std::make_pair(time, id));
+        auto it = s().m_timedCbs.find(std::make_pair(time, id));
         if (VL_LIKELY(it != s().m_timedCbs.end())) it->second.invalidate();
+        it = s().m_readWriteCbs.find(std::make_pair(time, id));
+        if (VL_LIKELY(it != s().m_readWriteCbs.end())) it->second.invalidate();
```

Comment:
Remove VL_LIKELY in whole function given we no longer have a single list we would expect it to be in.